.github/workflows/ci.yml

```diff
@@ -47,6 +47,14 @@ jobs:
         run: python ci/checks/check_no_line_number_snippets.py
       - name: name_tag_snippet_checker
         run: python ci/checks/check_only_name_tag_snippets.py
+      - name: integration_test_gets_run_checker
+        run: python ci/checks/check_integration_test_gets_run.py
+      - name: name_tag_snippet_referenced_checker
+        run: python ci/checks/check_name_tag_snippets_referenced.py
+      - name: run docs_snippet_checker
+        run: |
+          yarn install
+          python ci/checks/validate_docs_snippets.py
```

Comment:
We have a different test stage called `docs-snippets` which calls `invoke docs-snippet-tests`. This `doc-checks` stage calls `validate_docs_snippets.py`.  A few questions:
1. Do these tests belong in `docs-snippets`?
2. What are the difference between these tests?
It seems confusing to know where to put tests since they have such similar sounding names.

Reply:
1. Now that you mention it, I think so. I'll make the change
2. This ensures that no stray snippet opening/closing tags are present in our production docs